### ISSUE TITLE
Readme File should be updated

### ISSUE THREAD

**Issue by neeraj-2** [52North/52north.github.io#22]
**Readme** file should be updated so that any contributor can run in its local environment. All important **commands** should be mentioned.

**Comment by AtharvaParkale** [comment]
Can i work on this issue?

**Comment by Palaksharma23** [comment]
Yes all commands must present in readme , it will help new user to set it up on their local environment easily  .
